docs/adr/2025-10-17-using-go.md

```
@@ -0,0 +1,219 @@
+**Examples:**
+- PDF Generator: Go (runs in 100+ clusters, infrastructure-focused)
+- Studio Gateway: Go (Kubernetes API integration, runs in 100+ clusters)
+- Studio Operator: Go (Kubernetes operator pattern, infrastructure-focused, runs in 100+ clusters)
+- Studio Backend API: .NET (complex domain logic, control plane)
+- App Backend libraries: .NET (domain logic, shared libraries with apps, control plane and other platform services/products)
+
+## Appendix A: Resource usage comparison
+
+Comparison of baseline resource usage in local kind Kubernetes cluster using .NET NativeAOT and Go for a simple HTTP service responding to `/hello`:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)